include/odp/api/spec/ipsec.h

```diff
@@ -771,6 +771,20 @@ typedef struct odp_ipsec_sa_param_t {
 			 */
 			odp_cos_t dest_cos;
 
+			/** Enable reassembly of pre-IPsec IP fragments
+			 *
+			 *  When enabled, inline reassembly of pre-IPsec fragmented
+			 *  IP packets would be attempted. Reassembly attempt would
+			 *  be done following IPSEC inbound processing.
+			 *
+			 *  IPSEC configuration should enable reassembly to have this
+			 *  feature supported.
+			 *
+			 *  @see odp_ipsec_config()
+			 *
+			 */
+			odp_bool_t reassem_en;
```

Comment:
I think this commit should be squashed into the previous one since both have parts of IPsec reassebly config in them. Or alternatively, the IPsec specific part of the previous commit should be moved here.

Suggested rewording for the reassem_en description:

```
	/** Enable reassembly of IPsec tunneled fragments
	 *
	 *  Attempt reassembly of fragments after IPsec tunnel decapsulation.
	 *
	 *  Reassembly is attempted for inline or asynchronously processed packets,
	 *  not for packets processed using the synchronous API function.
	 *
	 *  Fragments received through different SAs will not be reassembled into
	 *  the same packet.
	 *
	 *  IPsec statistics reflect IPsec processing before reassembly and thus
	 *  count all individual fragments.
	 *
	 *  Reassembly may be enabled for an SA only if reassembly was enabled
	 *  in the global IPsec configuration.
	 *
	 *  @see odp_ipsec_config()
	 *
	 */
```

Reply:
I'll squash to the previous one. Later we can consider splitting based on patch size. Agreed on wording.